datasets/san_francisco_trees/street_trees/pipeline.yaml

```diff
@@ -0,0 +1,176 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+
+  - type: bigquery_table
+    table_id: street_trees
+
+    description: "San Francisco street trees table"
+
+dag:
+  airflow_version: 1
+  initialize:
+    dag_id: street_trees
+    default_args:
+      owner: "Google"
+
+      depends_on_past: False
+      start_date: '2021-03-01'
+    max_active_runs: 1
+    schedule_interval: "@daily"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "KubernetesPodOperator"
+
+      description: "Run CSV transform within kubernetes pod"
+
+      args:
+
+        task_id: "street_trees_transform_csv"
+
+        startup_timeout_seconds: 600
+
+        name: "street_trees"
+
+        namespace: "default"
+
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+                - matchExpressions:
+                    - key: cloud.google.com/gke-nodepool
+                      operator: In
+                      values:
+                        - "pool-e2-standard-4"
+
+
+        image_pull_policy: "Always"
+
+        image: "{{ var.json.san_francisco_trees.container_registry.run_csv_transform_kub }}"
+
+        env_vars:
+          SOURCE_URL: "https://data.sfgov.org/api/views/tkzw-k3nq/rows.csv"
+          SOURCE_FILE: "files/data.csv"
+          TARGET_FILE: "files/data_output.csv"
+          TARGET_GCS_BUCKET: "{{ var.value.composer_bucket }}"
+          TARGET_GCS_PATH: "data/san_francisco_trees/street_trees/data_output.csv"
+          PIPELINE_NAME: "street_trees"
+          CSV_HEADERS: >-
+            ["tree_id","legal_status","species","address","site_order","site_info","plant_type","care_taker","care_assistant","plant_date","dbh","plot_size","permit_notes","x_coordinate","y_coordinate","latitude","longitude","location"]
+          RENAME_MAPPINGS: >-
+            {"TreeID" : "tree_id" ,"qLegalStatus" : "legal_status" ,"qSpecies" : "species" ,"qAddress" : "address" ,"SiteOrder" : "site_order" ,"qSiteInfo" : "site_info" ,"PlantType" : "plant_type" ,"qCaretaker" : "care_taker" ,"qCareAssistant" : "care_assistant" ,"PlantDate" : "plant_date" ,"DBH" : "dbh" ,"PlotSize" : "plot_size" ,"PermitNotes" : "permit_notes" ,"XCoord" : "x_coordinate" ,"YCoord" : "y_coordinate" ,"Latitude" : "latitude" ,"Longitude" : "longitude" ,"Location" : "location"}
```

Comment:
As explained in a comment in another PR:

https://github.com/GoogleCloudPlatform/public-datasets-pipelines/pull/174#discussion_r717929113

You can use the `json` package to parse multiline string inputs (even with `\n` in them) into Python dicts.

Reply:
As discussed, we are going with this as of now . Hence , not changing...